config/formats/typescript-export-defition.ts

```diff
@@ -0,0 +1,145 @@
+
+import StyleDictionary from 'style-dictionary';
+import { format } from "prettier";
+const fs = require('fs');
+const path = require('path');
+
+const { fileHeader } = StyleDictionary.formatHelpers;
+
+/**
+ * unquoteTypes
+ * @description extract types from designTokenTypes (string with type definitions from file) and replace quoted types "Color" with unquoted Color
+ * @param output 
+ * @param designTokenTypes 
+ * @returns string with unquoted types
+ */
+const unquoteTypes = (output: string, designTokenTypes: string[]): string => {
+  // regex to match types like `type Color`
+  // const findTypesRegex = `type ([A-Za-z]+)`
+  // // extract types from designToken
+  // const types = [...designTokenTypes.matchAll(new RegExp(findTypesRegex, "g"))].map(match => match[1])
+  // join types for replacement
+  const regex = `"(${designTokenTypes.join('|')})"`
+  // remove strings
+  return output.replace(new RegExp(regex, "g"), '$1')
+}
+
+/**
+ * getTokenType
+ * @description extract content from token files
+ * @param path string
+ */
+const getTokenType = (tokenTypesPath: string): string => {
+  const designTokenType = fs.readFileSync(
+    path.resolve(tokenTypesPath), { encoding: 'UTF-8' }
+  );
+  //
+  return designTokenType
+}
+
+/**
+ * convertPropToType
+ * @description return take type attribute and return correct typescript type as string 
+ * @param type 
+ * @returns 
+ */
+const convertPropToType = (value: any, type: string): string => {
```

Comment:
Would value not always be string or number?

Reply:
No, it could be an object, e.g. for shadow. Although by this time it may already be converted to a string, I have to check this.

Reply:
Okay, I tested and at this point it is already a string, so I changed it, great catch. 👍